### PR TITLE
fix default on slider and form

### PR DESCRIPTION
**Pull request by Sylvainlfe**

### CODE REVIEW

**Review by Elea641 — APPROVED** [review]

File: client/src/components/Subscription.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              type="button"
+            >
+              PROFESSIONNEL
+            </button>
+          </li>
+        </ul>
+        <form className="flex flex-col items-center w-full h-full px-4 border border-pc rounded-b-lg">
+          <h2 className="font-bold my-10">INSCRIPTION</h2>
+          {fields.map((info) => (
+            <fieldset key={info.id} className="relative w-full pb-10">
```
Comment (Yoann-Fortin) [inline client/src/components/Subscription.jsx:48]: Why use a fieldset tag here ?

File: client/src/pages/SubscriptionPage.jsx [hunk 2]
```diff
@@ -78,64 +79,28 @@ function SubscriptionPage() {
     handleClickCustomer();
   }, []);
 
+  const customerButton = `w-full h-full p-0 rounded-none rounded-tl-lg cursor-pointer ${
```
Comment (Yoann-Fortin) [inline client/src/pages/SubscriptionPage.jsx]: Too much duplicate code with the following class

**Review by FabienRollet — APPROVED** [review]

File: client/src/components/Subscription.jsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              className={professionalButton}
+              type="button"
+            >
+              PROFESSIONNEL
+            </button>
+          </li>
+        </ul>
+        <form className="flex flex-col items-center w-full h-full px-4 border border-pc rounded-b-lg">
+          <h2 className="font-bold my-10">INSCRIPTION</h2>
+          {fields.map((info) => (
```
Comment (FranckWebPro) [inline client/src/components/Subscription.jsx:47]: info should be nammed singular field

File: client/src/components/Subscription.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            <fieldset key={info.id} className="relative w-full pb-10">
+              <input
+                type={info.type}
+                id={info.id}
+                name={info.text}
+                value={formValues[info.id]}
+                onChange={handleChange}
+                className="border-b-2 border-dc py-1 focus:border-b-2 focus:border-pc transition-colors focus:outline-none bg-inherit w-full"
```
Comment (FranckWebPro) [inline client/src/components/Subscription.jsx]: pseudo class/ element should be at the end of the breakpoint

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Pull request merged**